test/unit/test_fuzz_empty_data.py

```diff
@@ -0,0 +1,14 @@
+import os
+import tempfile
+from scripts.test import shared
+from . import utils
+
+
+class EmptyDataFuzzTest(utils.BinaryenTestCase):
```

Comment:
This is another example of a place where it would be great to have `gtest` set up :(

Reply:
How would gtest help this case? This case doesn't need things like asserts, but it's rather we check if the process crashes or not. But I have limited knowledge on gtest so there can be features I'm not aware of..

Reply:
yeah, I think gtest can test for crashes or their absence as well.